index.html

```diff
@@ -9,7 +9,12 @@
 <body>
     <h1>Pick an animal to adopt</h1>
     <ul>
-        <li>Dog</li>
+        <li>Dogs</li>
```

Comment:
hmm it might be a good idea to put this into a table in a future piece of work to more organize cats, and dogs that we have